odoo/addons/base/models/res_users.py

```diff
@@ -2240,8 +2227,7 @@ def change_password(self):
 )
 
 
-# pylint: disable=E0102
-class ResUsers(models.Model):  # noqa: F811
+class APIKeysUser(models.Model):
```

Comment:
Why do you already rename these?

Reply:
I explain it in the commit and pull request description

> While doing so, the problem regarding the re-definition of the same class name ResUsers and the call to super is demonstrated with super(ResUsers, demoted_users).write(vals): An infinite loop occurs because it calls super on the last defined ResUsers class rather than the current one.